packages/dds/tree/src/test/simple-tree/utils.ts

```diff
@@ -3,53 +3,59 @@
  * Licensed under the MIT License.
  */
 
-import { createIdCompressor } from "@fluidframework/id-compressor";
-import { MockFluidDataStoreRuntime } from "@fluidframework/test-runtime-utils";
+import { FlexFieldSchema } from "../../feature-libraries/index.js";
+import { FlexTreeView } from "../../shared-tree/index.js";
 import {
 	ImplicitFieldSchema,
 	TreeConfiguration,
 	TreeFieldFromImplicitField,
 	InsertableTreeFieldFromImplicitField,
+	toFlexConfig,
+	WrapperTreeView,
 } from "../../simple-tree/index.js";
-import { TreeFactory } from "../../treeFactory.js";
-import { typeboxValidator } from "../../external-utilities/index.js";
-import { ForestType } from "../../shared-tree/index.js";
+// eslint-disable-next-line import/no-internal-modules
+import { getProxyForField } from "../../simple-tree/proxies.js";
+import { flexTreeViewWithContent, flexTreeWithContent } from "../utils.js";
 
 /**
  * Given the schema and initial tree data, returns a hydrated tree node.
+ *
+ * For minimal/concise targeted unit testing of specific simple-tree content.
  */
 export function getRoot<TSchema extends ImplicitFieldSchema>(
 	schema: TSchema,
 	initialTree: () => InsertableTreeFieldFromImplicitField<TSchema>,
 ): TreeFieldFromImplicitField<TSchema> {
 	const config = new TreeConfiguration(schema, initialTree);
-	const factory = new TreeFactory({
-		jsonValidator: typeboxValidator,
-		forest: ForestType.Reference,
-	});
-	const tree = factory.create(
-		new MockFluidDataStoreRuntime({
-			idCompressor: createIdCompressor(),
-		}),
-		"tree",
-	);
-	const root = tree.schematize(config).root;
-	return root;
+	const flexConfig = toFlexConfig(config);
+	const tree = flexTreeWithContent(flexConfig);
+	return getProxyForField(tree) as TreeFieldFromImplicitField<TSchema>;
 }
 
 /**
- * Similar to JSON stringify, but preserves `undefined` and numbers numbers as-is at the root.
+ * Given the TreeConfiguration, returns a view.
+ *
+ * This works a much like the actual package public API as possible, while avoiding the actual SharedTree object.
+ * This should allow realistic (app like testing) of all the simple-tree APIs.
  */
+export function getView<TSchema extends ImplicitFieldSchema>(
+	config: TreeConfiguration<TSchema>,
+): WrapperTreeView<TSchema, FlexTreeView<FlexFieldSchema>> {
+	const flexConfig = toFlexConfig(config);
+	const view = flexTreeViewWithContent(flexConfig);
+	return new WrapperTreeView<TSchema, FlexTreeView<FlexFieldSchema>>(view);
+}
 
-export function pretty(arg: unknown): number | undefined | string {
+/**
+ * Similar to JSON stringify, but allows `undefined` at the root and returns numbers as-is at the root.
+ */
+
+export function pretty(arg: unknown): number | string {
 	if (arg === undefined) {
 		return "undefined";
 	}
 	if (typeof arg === "number") {
 		return arg;
 	}
-	if (typeof arg === "string") {
```

Comment:
This case is redundant with the case below which handles strings, with the exception that this case doesn't do escaping. Given that this is quoting the string and the docs don't mention lack of escaping, this seems like a bug and not a feature.